src/hooks/useLocalizedConstants.ts

```diff
@@ -0,0 +1,37 @@
+import { useApiConstants } from 'src/components/Constants/UseApiConstants';
+import {
+  ContactFilterStatusEnum,
+  PhaseEnum,
+  StatusEnum,
+} from 'src/graphql/types.generated';
+
+export const useLocalizedConstants = () => {
+  const constants = useApiConstants();
+
+  const getLocalizedContactStatus = (
```

Comment:
These should probably also be wrapped in `useCallback` with a dependency array of `[constants]`.